src/server/protocol.ts

```diff
@@ -442,6 +442,12 @@ namespace ts.server.protocol {
           * Errorcodes we want to get the fixes for.
           */
         errorCodes?: number[];
+
+        /**
+         * An optional set of settings to be used when computing indentation.
+         * If argument is omitted - then it will use settings for file that were previously set via 'configure' request or global settings.
+         */
+        options?: EditorSettings;

```

Comment:
VSCode will need to make sure that the formatting options are sent before they ask for the quick fix, just like they do with formatting.